src/components/PDFBuilderComponents/PDFDescription.jsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import styled from "@react-pdf/styled-components";
+import { Text, View, StyleSheet } from "@react-pdf/renderer";
+import { convertFromRaw, convertToRaw, EditorState } from "draft-js";
+
+const styles = StyleSheet.create({
+  bold: {
+    fontFamily: "Helvetica-Bold",
+  },
+  underline: {
+    textDecoration: "underline",
+  },
+  italic: {
+    fontFamily: "Times-Italic",
+  },
+  "header-one": {
+    fontSize: 18,
+  },
+  "header-two": {
+    fontSize: 16,
+  },
+  "header-three": {
+    fontSize: 14,
+  },
+  "header-four": {
+    fontSize: 12,
+  },
+  "header-five": {
+    fontSize: 10,
+  },
+  "header-six": {
+    fontSize: 8,
+  },
+});
+const TextArea = styled.Text`
+  font-size: 8px;
+  width: 280px;
+  margin-top: 6px;
+  font-weight: bold;
+  font-family: "Helvetica";
+`;
+
+const generateInlineStyle = (block) => {
+  const inlineStyleRanges = block.inlineStyleRanges;
+  const text = block.text;
+  let nestedTexts = [];
+  let offset = 0;
+  if (inlineStyleRanges.length === 0) return block.text;
```

Comment:
Also up to you, but I feel that having one-liner if statements can be hard to read and easy to miss.

```suggestion
  if (inlineStyleRanges.length === 0) {
     return block.text
  }
```